src/encryption.ts

```diff
@@ -1,7 +1,7 @@
 import { fromBase64url, toBase64url } from './utils'
 
 interface JWERecipient {
-  encrypted_key?: string // eslint-disable-line @typescript-eslint/camelcase
+  encrypted_key?: string // eslint-disable-line
```

Comment:
Why did you remove the rule that was being ignored here? Now I guess all rules are ignored?

Reply:
I was getting errors about it being missing, deprecated: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/camelcase.md

I can put it back but I think this might be a new typescript-eslint thing, and I don't really know how to control the new `naming-convention` rule inline like this -- got any suggestions?

Reply:
Was mostly curious. What happens if you remove the `eslint-disable-line` entirely?

Reply:
nothing! have removed it 🤞